src/Lean/MetavarContext.lean

```diff
@@ -1170,7 +1179,7 @@ mutual
         if (← read).mvarIdsToAbstract.contains mvarId then
           return mkAppN f (← args.mapM (visit xs))
         else
-          return (← elimMVar xs mvarId args).1
+          return (← elimMVar xs mvarId args true).1
```

Comment:
Please write `(usedLetOnly := true)` instead of `true`, and add comment explaining why it is set to `true` here.